prow/config.yaml

```diff
@@ -1,4 +1,13 @@
 presets:
+  - labels:
+      preset-gc-envs: "true"
```

Comment:
think about renaming this preset, maybe: preset-gc-project-compute-envs or split it to 2 presets:
- preset-gc-project-env
- preset-gc-compute-envs